src/components/navbar.js

```diff
@@ -0,0 +1,16 @@
+import { Link } from 'react-router-dom';
+import './navbar.css';
+
+const Navbar = () => (
+  <nav className="nav">
+    <div>
+      <Link to="/" className="header">Bookstore</Link>
+    </div>
```

Comment:
- [ ] [OPTIONAL] Kindly notice that there is no need for the `Bookstore` link as it directs to the page as the `Books` link does so kindly remove it.

Reply:
Thanks, I will do that now